examples/drivenet/drivenet_train.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to DriveNet\n",
+    "\n",
+    "DriveNet is a CNN able to drive an agent on a specific route. Among its features, DriveNet can follow the midlane and respect traffic lights.\n",
+    "\n",
+    "### Imitation Learning and Its limitations\n",
+    "DriveNet is trained using imitation learning on our Lyft L5 Prediction Dataset 2020. We feed examples of real driving experience to the model and we expect it to take the same actions as the driver did in those episodes. This is very similar to how tasks like classification are usually solved.\n",
+    "\n",
+    "Imitiation Learning is powerful, but it has a strong limitation. Even with a lot of data, is easy to drift away from the original train distribution. This may not be an issue for single-shot task (like classification), but this limit becomes immediately clear for autoregressive predictions.\n",
+    "\n",
+    "After training DriveNet, we would like it to take full control of the AV and determine its future locations. This means that subsequent predictions will be use as future locations for the AV. \n",
```

Comment:
useD.

Maybe this is too much in our 'language' though? At this point, readers might not think of modifying future positions in a zarr etc -> just say DN controls AV or so?